plutus-metatheory/src/Untyped/CEK.lagda.md

```diff
@@ -405,30 +417,48 @@ BUILTIN' b {pt = pt} {pa = pa} bt with trans (sym (+-identityʳ _)) (∔2+ pt) |
 ival : Builtin → Value
 ival b = V-I b base
 
+extendStack : Stack → List Value → Stack
```

Comment:
overly generic name?

Reply:
Agree. Will change it to `pushValueFrames`.